### PR TITLE
travis: Explicitly set --with-valgrind

### PR DESCRIPTION
**Pull request by jonasnick**
Also remove CPPFLAGS=-DVALGRIND because that's redundant with when
configured with --enable-valgrind.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ compiler:
   - gcc
 env:
   global:
-    - WIDEMUL=auto  BIGNUM=auto  ENDOMORPHISM=no  STATICPRECOMPUTATION=yes  ECMULTGENPRECISION=auto  ASM=no  BUILD=check  EXTRAFLAGS=  HOST=  ECDH=no  RECOVERY=no SCHNORRSIG=no EXPERIMENTAL=no CTIMETEST=yes BENCH=yes ITERS=2
+    - WIDEMUL=auto  BIGNUM=auto  ENDOMORPHISM=no  STATICPRECOMPUTATION=yes  ECMULTGENPRECISION=auto  ASM=no  BUILD=check  ENABLE_VALGRIND=yes RUN_VALGRIND=no EXTRAFLAGS=  HOST=  ECDH=no  RECOVERY=no SCHNORRSIG=no EXPERIMENTAL=no CTIMETEST=yes BENCH=yes ITERS=2
```
Comment (luke-jr) [inline .travis.yml:20]: Should it run valgrind to be sure it worked?

**Review by luke-jr — COMMENTED** [review]
Nit: I think it would be better to rename `VALGRIND` in a separate commit

File: .travis.yml (hunk 1 above)
Comment (jonasnick) [reply .travis.yml:20]: It runs valgrind if `RUN_VALGRIND` is `yes` which happens in lines 39 and 40.

**Comment by jonasnick** [comment]
Split into two commits.

File: .travis.yml (hunk 1 above)
Comment (real-or-random) [reply .travis.yml]: ah sure, I missed that this is the global line

**Comment by sipa** [comment]
utACK 4eecb4d6ef6d4f18be8870a5929feb1dae376d15

**Comment by elichai** [comment]
ACK 4eecb4d6ef6d4f18be8870a5929feb1dae376d15

**Pull request merged**